indico/modules/events/registration/templates/display/_registration_summary_blocks.html

```diff
@@ -183,7 +183,7 @@
                                         {{ choice }} ({{ nights }} nights)
                                     {%- endtrans %}
                                 {% elif item.field_data.field.input_type == 'multi_choice' %}
-                                    {{ item.friendly_data | join(', ') }}
```

Comment:
Where is this handled now?

Reply:
This change does not seem to be correct:

![image](https://user-images.githubusercontent.com/179599/167606912-bc2bd409-a2a9-42b7-8954-f230219eb1de.png)

Reply:
This is a mistake (I thought about handling the accompanying persons in this else, but I changed it back and broke multichoice. Reverting it!